web_src/js/components/RepoActionView.vue

```diff
@@ -342,6 +341,16 @@ const sfc = {
       return await resp.json();
     },
 
+    async fetchRerun(url, hrefURL) {
+      const resp = await this.fetchPost(url);
+      const results = await resp.json();
+      if (results.errorMessage) {
+        showErrorToast(`${results.errorMessage}`);
```

Comment:
```suggestion
        showErrorToast(results.errorMessage);
```

No need to convert to string if it is already.